SDWebImage/Core/UIImage+Transform.m

```diff
@@ -756,18 +749,11 @@ - (nullable UIImage *)sd_blurredImageWithRadius:(CGFloat)blurRadius {
         if (inputRadius - 2.0 < __FLT_EPSILON__) inputRadius = 2.0;
         uint32_t radius = floor(inputRadius * 3.0 * sqrt(2 * M_PI) / 4 + 0.5);
         radius |= 1; // force radius to be odd so that the three box-blur methodology works.
-        int iterations;
-        if (blurRadius * scale < 0.5) iterations = 1;
-        else if (blurRadius * scale < 1.5) iterations = 2;
-        else iterations = 3;
```

Comment:
This logic may cause wrong buffer been output to user because we create CGImage using `input` buffer (2 will cause input-output been swapped). Always use three box-blur methodology above.